### DISCUSSION TITLE
I need help with a startup project

### DISCUSSION THREAD

**Discussion by ZykenJ** [community/community#134364] [Programming Help]
### Select Topic Area

Show & Tell

### Feature Area

Projects

### Body

I need help with a startup project. My company is committed to developing a glasses trial project. I need to create a small program (in WeChat) because it will be used in China. It needs to accurately identify human faces such as eyes, nose, mouth, facial contours, including skin color, and ensure that all processing (facial recognition, feature extraction, glasses rendering) can be done in real time. I hope someone can help me. If you are interested, you can contact me

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by Shreyas719** [comment]
Make your project using AI .

**Comment by mattrltrent** [comment]
Try [OpenCV](https://opencv.org/about/) in Python.

**Comment by samus-aran** [comment]
Welcome to the GitHub Community, @ZykenJ , we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
